src/main/java/org/jitsi/jicofo/JitsiMeetGlobalConfig.java

```diff
@@ -103,7 +128,22 @@ private void init(ConfigurationService configService)
     {
         this.maxSSRCsPerUser
             = configService.getInt(
-                MAX_SSRC_PER_USER_CONFIG_PNAME, DEFAULT_MAX_SSRC_PER_USER);
+                    MAX_SSRC_PER_USER_CONFIG_PNAME, DEFAULT_MAX_SSRC_PER_USER);
+
+        jibriPendingTimeout
+            = configService.getInt(
+                    PENDING_TIMEOUT_PROP_NAME, DEFAULT_PENDING_TIMEOUT);
+
+        if (jibriPendingTimeout > 0)
+        {
+            logger.info(
+                    "Jibri requests in PENDING state will be timed out after: "
```

Comment:
It may be a good idea to change the log text to something like "The jibri pending timeout has been set to: ", because the above message can be confusing and we can think that the countdown has already started when we see it.